src/styles.css

```
@@ -707,3 +707,93 @@ footer a:hover{text-decoration:underline}
+  perspective: 1200px;
+  transition: transform 0.25s cubic-bezier(.2,.8,.2,1),
+              box-shadow 0.25s ease,
+              border-color 0.25s ease;
+  will-change: transform;
+}
+
+/* Main 3D hover */
+.org-card:hover {
+  transform: translateY(-10px) rotateX(10deg) rotateY(-10deg) scale(1.02);
+  box-shadow:
+    0 25px 60px rgba(0, 0, 0, 0.18),
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)